frontend/app/src/types/api/chains.ts

```diff
@@ -1,14 +1,17 @@
 import { z } from 'zod';
 
 const BasicChainInfo = z.object({
+  id: z.string(),
   name: z.string(),
   type: z.string()
 });
 
 const EvmChainInfo = z.object({
+  id: z.string(),
   name: z.string(),
   type: z.literal('evm'),
-  evmChainName: z.string()
+  evmChainName: z.string(),
+  nativeAsset: z.string()
```

Comment:
Is the native asset always set? at least from what we discussed it is only supposed to be there if it is not the same as the id.